Library/Homebrew/test/cmd/outdated_spec.rb

```diff
@@ -4,8 +4,11 @@
       setup_test_formula "testball"
       (HOMEBREW_CELLAR/"testball/0.0.1/foo").mkpath
 
+      setup_test_formula "foo"
+      (HOMEBREW_CELLAR/"foo/0.0.1/foo").mkpath
+
       expect { brew "outdated" }
-        .to output("testball\n").to_stdout
+        .to output("foo\ntestball\n").to_stdout
```

Comment:
Did you check if this specific case actually fails without `sort`?

Reply:
Yes. Commenting out the `sort` line results in the following failure:

```
1) brew outdated quiet output prints outdated Formulae
     Failure/Error:
       expect { brew "outdated" }
         .to output("foo\ntestball\n").to_stdout
         .and not_to_output.to_stderr
         .and be_a_success
     
       expected block to output "foo\ntestball\n" to stdout, but output "testball\nfoo\n"
```

Reply:
I assume that's locally but not on CI.

Reply:
Yes, locally.

CI should never fail as the tests were committed with the changes. I assume, unless I am misunderstanding something.

Reply:
I just mean that on CI, which is using HFS+, it probably always passes regardless of the sort, whereas locally it fails without the sort because you're using APFS. I'm not sure there's anything to be done about that really in terms of making the test actually test something on CI but I don't think we need to worry about `sort` not working.

Reply:
Good insight on CI using HFS+, thanks.

I originally tried making the the test case more expansive, with different formulae inserted in different orders. I simplified it as I wasn't sure the expanded test case provided any value.

Reply:
> I don't think we need to worry about `sort` not working.

Well, the test is not for `sort` itself, but for the fact that the output has to be sorted. Anyways, as long as it fails locally on APFS, it is good enough.